components/hook-http/node/index.test.mjs

```diff
@@ -0,0 +1,97 @@
+import { assertDeepEqual, assertEqual } from "../../__fixture__.mjs";
+import { Buffer } from "buffer";
+import { Writable, Readable } from "stream";
+import { buildTestDependenciesAsync } from "../../build.mjs";
+import HookHttp from "./index.mjs";
+
+const dependencies = await buildTestDependenciesAsync(import.meta.url);
+const {
+  parseContentType,
+  parseJSONSafe,
+  decodeSafe,
+  spyReadable,
+  spyWritable,
+} = HookHttp(dependencies);
+
+//////////////////////
+// parseContentType //
+//////////////////////
+
+assertDeepEqual(parseContentType("type/subtype; param=value"), {
```

Comment:
I don't think space is allowed here after the semicolon; same applies to other examples.

Reply:
From the spec: https://www.rfc-editor.org/rfc/rfc9110.html#name-media-type There is the example: `text/html; charset="utf-8"`. Anyway, even if spaces were illegal, it would not hurt to be more lenient that the spec?

Reply:
You're right. I actually double checked it and meant to remove this comment but either I didn't or it didn't work. Anyway, I was wrong, sorry :)